utils/clients/admin.ts

```diff
@@ -0,0 +1,16 @@
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import nodeFetch from "node-fetch";
+import { createClient } from "../../generated/api-admin/client";
+
+export const adminBaseUrl = getRequiredStringEnv("IO_FUNCTIONS_ADMIN_BASE_URL");
+export const adminToken = getRequiredStringEnv("IO_FUNCTIONS_ADMIN_API_TOKEN");
+// tslint:disable-next-line: no-any
+const fetchApi: typeof fetch = (nodeFetch as any) as typeof fetch;
```

Comment:
can we use a fetch with timeout like here:

https://github.com/pagopa/io-functions-app/blob/9cd4602b6a88192b86b762e7f355a986356ff238/utils/email.ts#L26 ?

Reply:
Fixed in [358e452](https://github.com/pagopa/io-functions-services/pull/68/commits/358e4526d63a6fa870cee31c280a912144c3b5ba)